includes/amp-helper-functions.php

```diff
@@ -301,6 +301,66 @@ function amp_add_frontend_actions() {
 	add_action( 'wp_head', 'amp_add_amphtml_link' );
 }
 
+/**
+ * Determine whether AMP is available for the current URL.
+ *
+ * @todo This should go into the AmpProject\AmpWP namespace?
+ * @since 1.6
+ *
+ * @return bool True if there is an AMP version, false otherwise.
```

Comment:
This made me first read as "true if there is an AMP version _number_", instead an "AMP version of the page".

Also, explaining the concepts of true and false and their exclusivity on a boolean value seems moot.

```suggestion
 * @return bool Whether there is an AMP version for the provided URL.
```

Reply:
Rather for the “current request” as opposed to the “provided URL”:

```suggestion
 * @return bool Whether there is an AMP version for the current request.
```